wasp/wasp.py

```diff
@@ -504,4 +507,27 @@ def schedule(self, enable=True):
 
         self._scheduling = enable
 
+
+    def set_theme(self, new_theme: bytes):
+        self._theme = array.array('H', new_theme)
+    def theme(self, theme_part: str) -> int:
+        """Returns the relevant part of theme. For more see ../tools/themer.py"""
+        theme_parts = ("ble",
+                       "scroll-indicator-up",
+                       "scroll-indicator-down",
+                       "battery-charging",
+                       "battery-empty",
+                       "battery-full",
```

Comment:
Similarly, why would charging and full be different colours.

Reply:
I'm not entirely sure battery-empty should be themeable either to be honest (the idea is that as the battery depletes the colour changes... and below a certain level the battery level gets too small to see so we colourize the frame with the colour the battery would be if we could see it).

Reply:
no problem, will rename "battery-charging" to "battery-charging-full" and will remove battery-full

Reply:
There's a problem here - battery-full and battery-charging *are* different colors on the watch right now

Reply:
so either we merge both to green by default, or we turn both into gray by default, (which causes problems with the fill on my device), or we keep them separate

or we could just set one of them to be fixed

Reply:
For now, I set it so that battery-full is always green, while battery-charging is themable.